src/test/visit_test_reports.py

```diff
@@ -487,7 +487,7 @@ def main():
         smtp_svr  = sys.argv[4]
         if len(sys.argv)  == 6:
             errors_only = bool(sys.argv[5])
-        email_summary(res,"visit-developers@ornl.gov",
+        email_summary(res,"brugger1@llnl.gov",
```

Comment:
@brugger1 didn't know exactly what to do with this one. It is setting the `from` field for the test results email I think. Since we're not using `visit-developers@ornl.gov` anymore, I set it to your LLNL email. A better choice might be LLNL's visit developers list? I honestly don't know if that is known outside LLNL in the sense that any VisIt developer hitting reply-all could successfully send an email to that address and whether its a good idea to have that email list public.

Reply:
That seems like the best option.